RecoEgamma/EgammaPhotonProducers/src/ReducedEGProducer.cc

```diff
@@ -238,6 +242,13 @@ ReducedEGProducer::ReducedEGProducer(const edm::ParameterSet& config)
           config.getParameter<edm::InputTag>("photonsPFValMap"))),
       gsfElectronPfCandMapT_(consumes<edm::ValueMap<std::vector<reco::PFCandidateRef>>>(
           config.getParameter<edm::InputTag>("gsfElectronsPFValMap"))),
+      recoHIPhotonIsolationMapInputToken_{
+          config.existsAs<edm::InputTag>("HIPhotonIsolationMapInput")
```

Comment:
other cases in this producer rely on empty inputTags; I suggest to stay with that logic, also considering that use of `existsAs` is generally not welcome.

Reply:
also, please stay with lower case 
```suggestion
          config.existsAs<edm::InputTag>("hiPhotonIsolationMapInput")
```

Reply:
Okay, I will drop the `existsAs` after the tests. I know it's not welcome because `fillDescriptions` is preferred. But since there is no fillDescriptions, isn't it better than an empty tag? I thought `existsAs` would make an eventual `fillDescription` migration easier because it makes it explicit that the parameter is optional. In the end I think the consistency with the other tags is the strongest argument for dropping the `existsAs` here, so I agree with you in the end.